zerver/lib/events.py

```diff
@@ -1061,6 +1061,13 @@ def apply_event(
     elif event["type"] == "update_global_notifications":
         assert event["notification_name"] in UserProfile.notification_setting_types
         state[event["notification_name"]] = event["setting"]
+    elif event["type"] == "user_settings":
+        if event["setting_name"] != "timezone":
```

Comment:
Let's add a comment documenting why this exception exists.

Reply:
Added the comment.